benchmarks/imagenet/vitb16/mae.py

```diff
@@ -123,7 +123,7 @@ def configure_optimizers(self):
         params, params_no_weight_decay = utils.get_weight_decay_parameters(
             [self.backbone, self.decoder]
         )
-        params.append(self.mask_token)
+        params.append(self.backbone.mask_token)
```

Comment:
params should already contain `self.backbone.mask_token` as we pass `self.backbone` to `utils.get_weight_decay_parameters`.